tests/by-util/test_printenv.rs

```diff
@@ -18,9 +18,11 @@ fn test_get_all() {
 fn test_get_var() {
     new_ucmd!()
         .env("KEY", "VALUE")
+        .env("FOO", "BAR")
         .arg("KEY")
         .succeeds()
-        .stdout_contains("VALUE\n");
+        .stdout_contains("VALUE\n")
+        .stdout_does_not_contain("BAR");
```

Comment:
I think it's clearer if you simply check for the exact expected output:
```suggestion
        .stdout_is("VALUE\n");
```

Reply:
Fixed it